app/HutchManager/Services/UserService.cs

```diff
@@ -35,9 +36,18 @@ public class UserService
   /// <param name="email">The email address to check</param>
   /// <returns></returns>
   public async Task<bool> CanRegister(string email)
-    => await _featureManager.IsEnabledAsync(Enum.GetName(FeatureFlags.AllowFreeRegistration)) ||
-        (await _db.RegistrationAllowlist.FindAsync(email) is not null);
-
+  {
+    return _registrationOptions.Registration == "free" ||
```

Comment:
couple of points on these comparisons:

1. Can we have an enum or a static class with constants for the possible registration options
2. When comparing, can we do a case-insensitive comparison, so that if someone puts "FREE" in their json files it still works 😋

if you use an enum for the options you can use `Enum.Parse<T>` and specify case insensitive. SargAssure has an example of this in the AddEmailSender extension

If you just use string constants, then do [this](https://stackoverflow.com/a/3694903)